pkg/api/peer_req_resp.go

```diff
@@ -18,6 +18,12 @@ type PeerAddReq struct {
 	MetaData  map[string]string `json:"metadata"`
 }
 
+// DeviceInfo represents structure in which devices are to be store in Peer MetaData
```

Comment:
Now that this is a separate package, this must be part of `plugins/device/api/resp.go`